app/Notifications/MailNotification.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\AnonymousNotifiable;
+use Illuminate\Notifications\Notification;
+
+class MailNotification extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    public function via(AnonymousNotifiable $notifiable): array
+    {
+        return ["mail"];
```

Comment:
You are not even using this `$notifiable` variable. I still don't get why it's `AnonymousNotifiable` used here.